hibernate-core/src/main/java/org/hibernate/engine/transaction/internal/TransactionImpl.java

```diff
@@ -31,29 +32,36 @@
 	private final TransactionCoordinator transactionCoordinator;
 	private final ExceptionConverter exceptionConverter;
 	private final JpaCompliance jpaCompliance;
+	private final AbstractSharedSessionContract session;
 
 	private TransactionDriver transactionDriverControl;
 
 	public TransactionImpl(
 			TransactionCoordinator transactionCoordinator,
 			ExceptionConverter exceptionConverter,
-			JpaCompliance jpaCompliance) {
+			JpaCompliance jpaCompliance,
```

Comment:
If we are passing Session, is there a reason to continue to pass along `JpaCompliance`?  We can just get that from `session.getFactory().getSessionFactoryOptions().getJpaCompliance()`

Reply:
no reason to pass `JpaCompliance`